projects/python/numpy.md

```diff
@@ -0,0 +1,11 @@
+
+# Numpy
+
+[![](https://img.shields.io/badge/Numpy-docs-green)](https://numpy.org)
+[![](https://img.shields.io/badge/Numpy-repo-blue)](https://github.com/numpy/numpy)
+
+NumPy is a fundamental package for scientific computing in Python. It provides the building blocks of linear algebra, statistics, and other numeric manipulation. In doing so it has become fundamental tool in both academic and industry settings.
+
+## Issues
+
+- [BUG: Inconsistent complex-->float casting failures](https://github.com/numpy/numpy/issues/23062)
```

Comment:
pick an issue for your project, don't 1-1 use my example

Reply:
Sorry! I thought we could use NumPy still if the issue was different.